components/ILIAS/Mail/tests/ilMailTaskProcessorTest.php

```diff
@@ -41,16 +41,16 @@ protected function setUp(): void
         parent::setUp();
 
         $this->languageMock = $this->getMockBuilder(ilLanguage::class)
-            ->disableOriginalConstructor()
-            ->getMock();
+                                   ->disableOriginalConstructor()
```

Comment:
We should try to avoid "Style" changes here and everywhere else to keep the PR smaller.

Reply:
This change did  not happen manually but through the recommended php fixer.

Reply:
Hmmm, when I execute `vendor/composer/vendor/friendsofphp/php-cs-fixer/php-cs-fixer fix --config=./scripts/PHP-CS-Fixer/code-format.php_cs --using-cache=no --dry-run -vvv components/ILIAS/Mail/classes` I only get one violation the class `ilMail`.

Reply:
That's odd. I will check what might have happened here.